userServiceSQLHelper.js

```diff
@@ -0,0 +1,28 @@
+function sortBy(sortBy) {
+    if (sortBy == "FirstName") {
```

Comment:
```suggestion
    return sortBy || "UserID"
```
That's it

Reply:
Fixed. But probably not exactly the way you suggested. I removed the "=== 'FirstName'" bit from the first line.
So now it's like this:
function sortBy(sortBy) {
    return sortBy || "UserID"
};